smartdevicelink/sdl_server
# Feature/sync app categories

Fixes #164 

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan
+ Check database to see new application categories retrieved from SHAID
+ Check database to see additional application category record attribute retrieved from SHAID (and existing records updated)
+ Confirm CRON tasks successfully execute

### Summary
Queries the SHAID API for application categories and performs an UPSERT command to the Postgres database to insert/update records as needed.

### Changelog
##### Breaking Changes
* No breaking changes

##### Enhancements
* Automatic application category syncing from SHAID

##### Bug Fixes
* Resolves errors when attempting to sync an application with a new category.

### Tasks Remaining:
None

### CLA
